bin/test.py

```diff
@@ -0,0 +1,282 @@
+#!/usr/bin/env python
+
+import boto3, botocore, os, time, sys
+from os.path import expanduser
+from botocore.client import Config
+
+TEST_APP_BUCKET_PREFIX='awslabs-startup-kit-templates-test-eb-v1-tmp-'
+TEST_APP_SOURCE_BUCKET='awslabs-startup-kit-templates-test-eb-v1'
+TEST_PYTHON_APP_KEY='eb-python-flask.zip'
+STACK_BUCKET='awslabs-startup-kit-templates-deploy-v1'
+EB_TEMPLATE_URL='https://s3.amazonaws.com/awslabs-startup-kit-templates-deploy-v1/vpc-bastion-eb-rds.cfn.yml'
+VPC_TEMPLATE_URL='https://s3.amazonaws.com/awslabs-startup-kit-templates-deploy-v1/vpc.cfn.yml'
+VPC_BASTION_TEMPLATE_URL='https://s3.amazonaws.com/awslabs-startup-kit-templates-deploy-v1/vpc-bastion.cfn.yml'
+KEY_PAIR_PREFIX='sktemplates-test-'
+
+# TODO Add support for Fargate test in supported regions
```

Comment:
I wonder if it makes sense to break this down into multiple scripts that reuse the same functions, each of which have a `SUPPORTED_REGIONS` list to make it really clear what is under test and where. If you're iterating on the Fargate templates, for example, it makes not much sense to run the Elastic Beanstalk acceptance tests.

Reply:
Possibly. Rich wants the foundation in and then we are going to iterate on it.

Reply:
Backlog SIM created